js_modules/dagit/packages/core/src/assets/AssetMaterializations.tsx

```diff
@@ -47,11 +47,11 @@ const LABEL_STEP_EXECUTION_TIME = 'Step Execution Time';
 
 /**
  * If the asset has a defined partition space, we load all materializations in the
- * last 200 partitions. This ensures that if you run a huge backfill of old partitions,
- * you still see accurate info for the last 200 partitions in the UI. A count-based
+ * last 100 partitions. This ensures that if you run a huge backfill of old partitions,
```

Comment:
I think 120 partitions are actually still being loaded for defined partition spaces, maybe this comment is out of date? This change only affects non-SDA partitioned assets

cc @bengotow

Reply:
Hoping to land this before the release tomorrow, to unblock some cloud users